cores/arduino/stm32/timer.c

```diff
@@ -560,8 +560,8 @@ uint32_t getTimerIrq(TIM_TypeDef* tim)
 void TimerHandleDeinit(stimer_t *obj)
 {
   if(obj != NULL) {
-    HAL_TIM_Base_DeInit(&(obj->handle));
```

Comment:
I think this should be fixed independently on this PR..

Reply:
We could include it, since @fpistm won't merge this right away. Can you write a patch?